spec/integration/init_web_spec.cr

```diff
@@ -133,3 +132,18 @@ private def compile_and_run_specs_on_test_project
     should_run_successfully "crystal spec"
   end
 end
+
+private def with_project_cleanup(project_directory = "test-project")
+  root_directory = FileUtils.pwd
+  yield
+  output = IO::Memory.new
+  Process.run(
+    "lucky db.drop",
+    env: ENV.to_h,
+    output: output,
+    shell: true
+  )
+  output.to_s.should contain("Done dropping")
+  FileUtils.cd root_directory
+  FileUtils.rm_rf project_directory
```

Comment:
This was moved from the `Spec.before_each` block. It should only need to run after any spec that creates the project directory.